client/src/containers/section/page/layers/layer.tsx

```diff
@@ -27,14 +31,89 @@ export default function Layer({ id, attributes }: LayerGroupLayersDataItem) {
     }
   };
 
-  const { title } = attributes || {};
+  const {
+    title,
+    description,
+    license,
+    citation,
+    cautions,
+    related_publications,
+    resource_contact_name,
+    resource_contact_url,
+    metadata_contact_name,
+    metadata_contact_url,
+    source,
+    source_url,
+  } = attributes || {};
+
+  interface Field {
+    key: string;
+    value: string;
+    url?: string;
+  }
+
+  const fields: Field[] = [
+    { key: 'License', value: license || '' },
+    { key: 'Citation', value: citation || '' },
+    { key: 'Cautions', value: cautions || '' },
+    { key: 'Related publications', value: related_publications || '' },
+    { key: 'Resource Contact', value: resource_contact_name || '', url: resource_contact_url },
+    { key: 'Metadata Contact', value: metadata_contact_name || '', url: metadata_contact_url },
+    { key: 'Source', value: source || '', url: source_url },
+  ].filter((f) => f.value !== '');
+
+  const renderField = ({ key, value, url }: Field) => {
+    if (!key || !value) return null;
+    return (
+      <div className="flex gap-10 text-sm">
+        <div className="w-[134px] min-w-[134px] font-semibold">{key}</div>
+        {url ? (
+          <a href={url} target="_blank" rel="noreferrer" className="font-semibold text-yellow-500">
+            {value}
+          </a>
+        ) : (
+          <div>{value}</div>
+        )}
+      </div>
+    );
+  };
+  const isActive = layers.includes(id);
   return (
-    <li key={id} className="space-y-2.5">
+    <li
+      key={id}
+      className={cn('mb-4 space-y-4 bg-yellow-50 p-6', { 'bg-slate-700 text-white': isActive })}
+    >
       <header className="flex justify-between space-x-2.5 py-1 pl-2">
-        <h4>{title}</h4>
-
-        <Switch checked={layers.includes(id)} onCheckedChange={handleLayerChange} color="yellow" />
+        <h4 className="font-serif text-lg leading-7">{title}</h4>
+        <div className="flex gap-4">
+          <Dialog>
+            <DialogTrigger asChild>
+              <Info
```

Comment:
You should put the icon in a button so that it can be focused and “clicked” via the keyboard. It'd be nice to have an alternative text as well (using a `<span />` with `sr-only` for example).